torch_xla/core/dynamo_bridge.py

```diff
@@ -345,4 +349,18 @@ def optimized_mod(*args):
     none_remover.add_nones(result)
     return result
 
-  return optimized_mod
+  return optimized_mod(*xla_args)
+
+def extract_compile_graph(xla_model, xla_args):
+  supported_ops = XlaOperatorSupport()
+  partitioner = CapabilityBasedPartitioner(xla_model, supported_ops)
+  partitions = partitioner.propose_partitions()
+  partitioned_graph = partitioner.fuse_partitions(partitions)
+
+  for node in partitioned_graph.graph.nodes:
+    if node.op == "call_module" and "fused_" in node.name:
+      fused_module = getattr(partitioned_graph, node.name)
+      fused_module._wrapped_call = extract_internal
```

Comment:
oh OK, I think this logic make more sense to me. We return multiple fused module and each of them will call `extract_internal` (maybe rename this function to  `extract_and_partition_graph ` and leave origional one `extract_compile_graph`). @wonjoolee95 I also left a comment in you origional pr, maybe you can also give this approach a shot since core logic is intact.

Reply:
Renaming will require corresponding changes in dynamo's backend file. I will note this and make the changes once everything else is fixed.